app/models/effective/datatables/employer_datatable.rb

```diff
@@ -179,6 +179,12 @@ def nested_filter_definition
         end
         filters
       end
+
+      def authorized?(current_user, _controller, _action, _resource)
+        return false unless current_user
+
+        HbxProfilePolicy.new(current_user, nil).employer_index?
```

Comment:
This method is just checking if 'hbx_staff' exists in the roles field of the User and does not match with what we have in the controller action(Exchanges::HbxProfilesController#employer_poc).

```suggestion
        current_user&.has_hbx_staff_role?
```